### ISSUE TITLE
Colouring in Git commit editor a bit off

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#153678]
Testing #153366

I would expect the modified lines to be in green, since they are commented out and will be ignored?

![Screenshot 2022-06-29 at 14 27 54](https://user-images.githubusercontent.com/1926584/176436210-e1c2914e-e6ed-4e15-a7e6-ae349ca82245.png)

**Comment by joaomoreno** [comment]
@isidorn I'm not sure, but that might be because `added`, `modified` and `deleted` lines appear differently, no?

**Comment by joaomoreno** [comment]
Exactly: https://github.com/microsoft/vscode/blob/1df4b3ceb899d54063793463ead4baea009d5fa5/extensions/git-base/syntaxes/git-commit.tmLanguage.json#L89-L122
